methods.py

```diff
@@ -7,6 +7,9 @@
 import datetime
 import subprocess
 from compat import iteritems, isbasestring
+import SCons.Util
```

Comment:
Since those are only used in `get_libs_full_path`, you can add them at the start of the function directly.

Reply:
Good point; will fix that.

Reply:
Fixed